bin/tape

```diff
@@ -1,12 +1,34 @@
 #!/usr/bin/env node
 
-var path = require('path');
+var resolveModule = require('resolve').sync;
+var resolvePath = require('path').resolve;
+var parseOpts = require('minimist');
 var glob = require('glob');
 
-process.argv.slice(2).forEach(function (arg) {
+var opts = parseOpts(process.argv.slice(2), {
+        alias: { 'r': 'require' },
+        string: 'require',
+        default: {'r': [] }
+    });
+
+var cwd = process.cwd();
+
+if (typeof opts.require === 'string') {
```

Comment:
I did something similar [previously][1], but @Raynos suggested the change to the `typeof` block instead. I don't really care much either way to be honest.

[1]: https://github.com/substack/tape/pull/224#discussion-diff-48197171R19